go/pkg/sdk/sdk_test.go

```diff
@@ -26,42 +27,45 @@ import (
 )
 
 type testCase struct {
-	rpcEndpoint   string
-	chainId       int64
-	vault         string
-	vaultAddress  common.Address
-	vaultDecimals int64
-	stable        constants.Stablecoin
-	stableAddress common.Address
-	sDai          common.Address
-	testAddress   common.Address
-	privateKeyHex string
+	rpcEndpoint    string
```

Comment:
I've just seen that we don't use the linter in a github action.
Could you add a github action to do so? Basically, we just run the golanci-lint as pre-commit hook and in the CI.
If we have both, it will make sure that we don't have such "indentation changes" in a pull request.